repositories/MerchOrderRepository.ts

```diff
@@ -69,7 +69,9 @@ export class OrderPickupEventRepository extends BaseRepository<OrderPickupEventM
   public async getFuturePickupEvents(): Promise<OrderPickupEventModel[]> {
     return this.getBaseFindQuery()
       .where('"end" >= :now')
+      .andWhere(':orderCount <= "orderLimit"')
       .setParameter('now', new Date())
+      .setParameter('orderCount', 0)
       .getMany();
   }
```

Comment:
Any reason we are setting 0 as the `orderCount` here? Future pickup events can have orders scheduled for them, so we'd need to filter by the actual order count to get only the future pickup events that have not reached their capacity yet in the "place order" page on the frontend. However, we need to also be able to get all future pickup events regardless of capacity in the admin page for admins to scroll through, so let's remove the filter here and just have the frontend perform the filtering on the 'place order' side to only display future events with capacity.

Reply:
Resolved in d149edc.